src/Microsoft.VisualStudio.Editor.Razor/DefaultEditorSettingsManagerInternal.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.CodeAnalysis.Razor.Editor;
+
+namespace Microsoft.VisualStudio.Editor.Razor
+{
+    internal class DefaultEditorSettingsManagerInternal : EditorSettingsManagerInternal
```

Comment:
Consider renaming this hierarchy to be called `WorkspaceEditorSettings`. That's a better name and makes the different between these two things clear.